app/views/users/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>Here is a list of users</h1>
\ No newline at end of file
+<div class="list">
+  <% @users.each do |user| %>
+    <%= link_to user do %>
+      <%= render partial: "user", object: user %>
+    <% end %>
+  <% end %>
+</div>
```

Comment:
### OPTIONAL
- [ ] To show all users you don't have to use each method

Example:
![image](https://user-images.githubusercontent.com/38635158/161261888-414c8940-01aa-4b79-b8a9-267afb1afe40.png)